content/applications/inventory_and_mrp/quality/quality_reporting/quality_checks_report.rst

```diff
@@ -0,0 +1,195 @@
+--> Reporting --> Quality Checks`.
+
+There are four main views that can be used to display quality check data:
+
+- :ref:`Graph view <quality_alerts_report/graph-view>`
+- :ref:`Pivot view <quality_alerts_report/pivot-view>`
+- :ref:`Kanban view <quality_alerts_report/kanban-view>`
+- :ref:`List view <quality_alerts_report/list-view>`
+
+By default, the dashboard displays data in the graph view, using the bar chart graph type. To select
+a different view, click on the corresponding button in the top-right corner of the dashboard.
+
+.. figure:: quality_checks_report/view-select.png
+   :align: center
+   :alt: The view select buttons on the quality checks report dashboard.
+
+   The view select buttons on the quality checks report dashboard. From left to right: Graph, Pivot,
+   Kanban, and List.
+
+.. _quality_alerts_report/graph-view:
+
+Graph view
+----------
+
+To display quality check data in graph view, click the :icon:`fa-area-chart` :guilabel:`(Graph)`
+button in the top-right corner of the dashboard.
+
+There are three chart types that can be used to visualize quality check data:
+
+- :ref:`Bar chart <quality_alerts_report/bar-chart>`
+- :ref:`Line chart <quality_alerts_report/line-chart>`
+- :ref:`Pie chart <quality_alerts_report/pie-chart>`
+
+By default, graph view displays data using the bar chart graph type. To select a different graph
+type, click the corresponding button in the top-left corner of the dashboard.
+
+.. figure:: quality_checks_report/graph-select.png
+   :align: center
+   :alt: The graph select buttons on the quality checks report dashboard.
+
+   The graph select buttons on the quality checks report dashboard. From left to right: Bar Chart,
+   Line Chart, and Pie Chart.
+
+.. _quality_alerts_report/bar-chart:
+
+Bar chart
+~~~~~~~~~
+
+To display quality check data in a bar chart, click the :icon:`fa-bar-chart` :guilabel:`(Bar Chart)`
+button in the top-left corner of the dashboard.
+
+The x-axis of the bar chart shows the three stages that a quality check can be organized under:
+:guilabel:`Failed`, :guilabel:`To do`, and :guilabel:`Passed`. The y-axis of the bar chart shows the
+number of quality checks in each stage, which is displayed by a bar above each stage name.
+
+.. image:: quality_checks_report/bar-chart.png
+   :align: center
+   :alt: The bar chart graph view on the quality checks report dashboard.
+
+.. _quality_alerts_report/line-chart:
+
+Line chart
+~~~~~~~~~~
+
+To display quality check data in a line chart, click the :icon:`fa-line-chart` :guilabel:`(Line
+Chart)` button in the top-left corner of the dashboard.
+
+The x-axis of the line chart shows the three stages that a quality check can be organized under:
+:guilabel:`Failed`, :guilabel:`To do`, and :guilabel:`Passed`. The y-axis of the line chart shows
+the number of quality checks in each stage, which is displayed by a point on the graph above each
+stage name. Each of these points is linked by a line.
```

Comment:
again, might be beneficial to briefly mention that this is just the default setting, and other options are available to expand on the analysis of quality checks

Reply:
that could even be mentioned somewhere near the top of the doc, because it *is* an all-encompassing statement about these reporting pages/views, in general. just a thought...